src/GUI/Pages/CreateProject/CreateProjectModalWindow.razor

```diff
@@ -0,0 +1,40 @@
+@using LT.DigitalOffice.GUI.Services.ApiClients.ProjectService
+
+<div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      
+      <div class="modal-header">
+        
+        <h5 class="modal-title" id="exampleModalLongTitle">Creation project</h5>
```

Comment:
1 kebab-case to id
2 Add "Creation project" to resx

Reply:
Fixed